src/AppInstallerCLIE2ETests/AppInstallerCLIE2ETests.csproj

```diff
@@ -21,7 +21,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.PowerShell.SDK" Version="7.2.8" />
+    <PackageReference Include="Microsoft.PowerShell.SDK" Version="7.2.18" />
     <PackageReference Include="Newtonsoft.Json" Version="13.0.2" />

```

Comment:
The build pipeline reported an error due to mismatch between the version of the other dependencies. `Microsoft.PowerShell.SDK` is requiring higher versions than what we have listed here. For example (not the only instance):
```
NU1605: Warning As Error: Detected package downgrade: Newtonsoft.Json from 13.0.3 to 13.0.2. Reference the package directly from the project to select a different version. 
 AppInstallerCLIE2ETests -> Microsoft.PowerShell.SDK 7.2.18 -> System.Management.Automation 7.2.18 -> Newtonsoft.Json (>= 13.0.3) 
 AppInstallerCLIE2ETests -> Newtonsoft.Json (>= 13.0.2)
```

You can review the build logs to see the versions needed, but you should also see it if you build the project locally.

Alternatively, instead of increasing the versions here, it may be possible to remove some of the other dependencies if we are not using them directly to let the version from the PS SDK be the one we use.

Reply:
> The build pipeline reported an error due to mismatch between the version of the other dependencies. `Microsoft.PowerShell.SDK` is requiring higher versions than what we have listed here. For example (not the only instance):
> ```
> NU1605: Warning As Error: Detected package downgrade: Newtonsoft.Json from 13.0.3 to 13.0.2. Reference the package directly from the project to select a different version. 
>  AppInstallerCLIE2ETests -> Microsoft.PowerShell.SDK 7.2.18 -> System.Management.Automation 7.2.18 -> Newtonsoft.Json (>= 13.0.3) 
>  AppInstallerCLIE2ETests -> Newtonsoft.Json (>= 13.0.2)
> ```
> 
> You can review the build logs to see the versions needed, but you should also see it if you build the project locally.
> 
> Alternatively, instead of increasing the versions here, it may be possible to remove some of the other dependencies if we are not using them directly to let the version from the PS SDK be the one we use.

I don't understand what you are saying please tell it step by step.